### PR TITLE
Recipe for "pysptools" package (Tools for hyperspectral imaging)

### PR DESCRIPTION
**Pull request by dturaev**
Recipe is based on "conda skeleton pypi pysptools". Dependencies were added, the "about" section was modified for better readability.

PySptools is a python module that implements spectral and hyperspectral algorithms. Specializations of the library are the endmembers extraction, unmixing process, supervised classification, target detection, noise reduction, convex hull removal and features extraction at spectrum level.

### CODE REVIEW

**Comment by dturaev** [comment]
Ok, now I'm stuck. The errors say that the "import pysptools.doc" test didn't work. However the build and all tests worked with conda build locally. I also ran "python setup.py install" directly and can import pysptools.doc afterwards. Please help?

**Comment by dturaev** [comment]
Ok, the problem was that the tests tried to import two directories that were not a module, "doc" and "examples". But what's the problem with appveyor?

**Comment by dturaev** [comment]
Ok, the cvxopt dependency was apparently not available on Windows. It was optional, so I just removed all optional dependencies, which makes more sense anyway.

**Comment by dturaev** [comment]
@conda-forge/core I'll be happy to modify the recipe if something's wrong with it?

File: recipes/pysptools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (CJ-Wright) [inline recipes/pysptools/meta.yaml]: Could this be noarch?

File: recipes/pysptools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - pysptools.tests
+    - pysptools.util
+
+about:
+  home: 'https://pysptools.sourceforge.io/'
+  license: Apache 2.0
+  license_family: APACHE
+  license_file: 'pysptools/LICENSE.txt'
+  summary: A hyperspectral imaging tools box
+  description: PySptools is a hyperspectral and spectral imaging library that provides spectral algorithms for the Python programming language. Specializations of the library are the endmembers extraction, unmixing process, supervised classification, target detection, noise reduction, convex hull removal and features extraction at spectrum level. For more information, see the docs.
```
Comment (CJ-Wright) [inline recipes/pysptools/meta.yaml]: Could you split this up over multiple lines?

Comment (dturaev) [reply recipes/pysptools/meta.yaml]: Thanx! Ok, done.

**Pull request closed without merging**